environment/src/main/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorService.java

```diff
@@ -73,6 +75,25 @@ public ValidationResultBuilder validateNetworkCreation(Environment environment,
         return networkCreationValidator.validateNetworkCreation(environment, network, subnetMetas);
     }
 
+    public ValidationResult validateParentChildRelation(Environment environment, String parentEnvironmentCrn) {
+        ValidationResultBuilder resultBuilder = new ValidationResultBuilder();
+
+        resultBuilder.ifError(() -> Objects.nonNull(parentEnvironmentCrn) && Objects.isNull(environment.getParentEnvironment()),
+                String.format("Active parent environment with crn '%s' is not available in account '%s'.", parentEnvironmentCrn, environment.getAccountId()));
```

Comment:
I think the name is more useful here

Reply:
Done in af20f701d8c3e87addc57f612c9b6fc642d0daf5